Felienne/hedy
# [FIX] Replace file structure of for-teachers page from .md to .yaml files

**Description**
Currently we depend on .md files for the content of several pages. For correctly retrieving and displaying sections several pre-process functions are written. However, the .md file structure gives us less freedom with front-end customization than we would like. By replacing the structure to depend on .yaml files (similar to how adventures are stored) we are better able to customize our UI dependent on the content. In this PR we replace the .md file structure by a .yaml file structure for content translations. A new function is written to retrieve the correct text and the front-end template page is adjusted to correctly display this content.

**Fix for**
This PR fixed #1389 (partly).

**How to test**
Navigate to for-teachers page and notice so difference on the front-end. On code base, notice the addition of a new folder for storing the .yaml files and verify the correct retrieval of translations.

**Checklist**

If you're unsure about any of these, don't hesitate to ask. We're here to help!
